data/navigations_de.yml

```
@@ -93,6 +94,11 @@
     - title: Jobs
       url: /jobs/index.html
       icon: icons/jobs.svg
+      not_main: true
```

I would go for:

```suggestion
      main: false
```

So defaults to true unless set the false.